assets/src/components/searchForm.tsx

```diff
@@ -246,6 +246,7 @@ export const Combobox = ({
   const [autocompleteEnabled, setAutocompleteEnabled] = useState(true)
 
   const onSubmit = (e: SyntheticEvent) => {
+    e.preventDefault()
```

Comment:
question: I believe this may have been a merge issue? based on our [previous conversation](https://github.com/mbta/skate/pull/2659#discussion_r1638850564) at least